storyboarding/scripting/compound_commands.rst

```diff
@@ -12,6 +12,405 @@ Loop
+
+  If you wonder why this is the case, the answer is that hitnormals work for everyone differently. Currently there is the option to configure the option ``LayeredHitSounds`` in your ``Skin.ini`` to 0 causing hitnormals no longer be played on notes that have additionals.
+  As this is a setting determined by the user in most cases, there is no way to get a consistent experience for all players without the already mentioned workaround anyway.
+
+Example: Hitsounds
+..................
+Aoba wants to play the Taiko drum! She is very inexperienced but maybe you can help her out?
+
+.. image:: img/compound_commands/aobaTaiko.png
+  :scale: 20%
+  :alt: Full of optimism: Aoba
+  :align: left
+
+.. image:: img/compound_commands/drum.png
+  :scale: 30%
+  :alt:  ...and the Taiko drum!
+  :align: right
+
+If you aren't familiar with Taiko mapping, no problem, we got that covered in a few words.
+
+There are 2 types of notes in Taiko, Don (red) and Kat (blue). If it has a whistle and/or clap hitsound attached it is a Kat, otherwise it is a Don note.
+
+
+We can make Aoba hit the drum by catching the respective trigger conditions:
+
+These would be ``HitsoundWhistle`` and ``HitsoundClap`` for Kat.
+
+For Don we have to create a workaround and give each note a separate Hitsoundset because we can't react to the hitnormal without reacting to the additionals as well.
+
+So let's say we put Hitsoundset 4 for each Don, meaning we have to specify the triggercondition as ``Hitsound4``.
+
+
+To make this as simple as possible in terms of example, we'll introduce 2 more versions of Aoba in which she is using one or the other drumstick to hit the Taiko.
+
+We will also put one of her pigtails into a differentsprite so that we can display the drumsticks in front of the drum but her hair behind it.
+In total we got 5 sprites:
+
+.. image:: img/compound_commands/aobaTaikoIdle.png
+  :scale: 20%
+  :alt: Aoba is idling
+  :align: left
+
+.. image:: img/compound_commands/aobaTaikoDon.png
+  :scale: 20%
+  :alt: Aoba is hitting a don
+  :align: left
+
+.. image:: img/compound_commands/aobaTaikoKat.png
+  :scale: 20%
+  :alt: Aoba is hitting a kat
+  :align: left
+
+.. image:: img/compound_commands/aobaHair.png
+  :scale: 20%
+  :alt: One of Aoba's pigtails
+  :align: left
+
+.. image:: img/compound_commands/drum.png
+  :scale: 30%
+  :alt: The taiko drum
+  :align: left
+
+First of all we are moving our static sprites into place:
+
+.. code-block:: c
+  :linenos:
+  :caption: Our static sprites
+  :name: Static sprites
+
+  Sprite,Background,Centre,"sb\aobaHair.png",192.625,140
+   S,0,0,,0.2
+   F,0,0,,1
+  Sprite,Background,CentreLeft,"sb\drum.png",186,163
+   S,0,0,,0.3
+   F,0,0,,1
+
+When nothing is happening the sprite of the idling Aoba should be shown. This will be the case at the start of the beatmap.
```

Comment:
Changed. Although I have to note that it sounds more cute when it's a sprite of Aoba instead of an Aoba sprite. More personal involvement from Aoba, you know ;)